main.py

```diff
@@ -1,10 +1,55 @@
 import init_django_orm  # noqa: F401
+import json
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as source_file:
+        players = json.load(source_file)
+
+    for key, data in players.items():
+        new_player = Player(
+            nickname=key,
+            email=data.get("email"),
+            bio=data.get("bio")
+        )
+
+        guild = data.get("guild")
+        if guild is not None:
+            guild_name = guild.get("name")
+            if not Guild.objects.filter(name=guild_name).exists():
+                Guild.objects.create(
+                    name=guild_name,
+                    description=guild.get("description")
+                )
+            guild_from_db = Guild.objects.get(name=guild_name)
```

Comment:
There is no need to query the database additionally in case you create a record, `.create()` will return the required object. You need to write it to a variable. Execute `.get()` in the `else` statement to achieve that.